packages/ui/src/components/SymbiosisBarChart/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+
+import { Chart } from "../Charts";
+import type { ChartConfig, ChartDataPoint } from "../Charts/types";
+import type { SymbiosisBarChartProps } from "./types";
+import { cn } from "../../utils/cn";
+
+const SymbiosisBarChart = ({
+  data,
+  config,
+  xAxisFormatter,
+  tooltipLabelFormatter,
+  className,
+  legendClassName,
+  tooltipClassName,
+}: SymbiosisBarChartProps) => {
+
+  const defaultTickFormatter = React.useCallback((value: string) => {
+    const date = new Date(value);
+    return date.toLocaleDateString("en-US", {
+      month: "short",
+      day: "numeric",
+    });
+  }, []);
+
+  const defaultTooltipLabelFormatter = React.useCallback((value: string) => {
+    const date = new Date(value);
+    return date.toLocaleDateString("en-US", {
+      month: "short",
+      day: "numeric",
+      year: "numeric",
+    });
+  }, []);
+
+  return (
+    <Chart.Container config={config} className={cn("aspect-auto h-[250px] w-full", className)}>
```

Comment:
[minor suggestion] Might need to make the `height` adjustable via props.

```
<Chart.Container config={config} className={cn(`aspect-auto h-[${height}px] w-full`, className)}>
```

Reply:
It's not a bad idea but we can just pass anything we need through the `className` prop.
Do you guys think we need another prop for the height? 🤔

Reply:
That's fair, probably `className` override is good enough here